keystone/meta/salt.yml

```diff
@@ -30,10 +30,10 @@ minion:
     {%- for profile_name, identity in client.get('server', {}).items() %}
       {%- set protocol = identity.admin.get('protocol', 'http') %}
 
-      {%- if identity.admin.get('api_version', '2') == '3' %}
+      {%- if identity.admin.get('api_version', '2') == 3 %}
```

Comment:
this depends on what you store in metadata, i prefer versions as strings, what do you think?

Reply:
or cast the identity.admin.get('api_version', '2') to string